flake.nix

```diff
@@ -0,0 +1,38 @@
+{
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs?ref=nixos-23.11";
+  };
+  outputs =
+    {
+      self,
+      nixpkgs,
+      flake-utils,
+      ...
+    }:
+    flake-utils.lib.eachDefaultSystem (
+      system:
+      let
+        pkgs = import nixpkgs { inherit system; };
+      in
+      {
+        devShells.default = pkgs.mkShell {
+          buildInputs = with pkgs; [
+            # generic
+            git
+            git-lfs
+            gnumake
+            gnutar
+
+            # frontend
+            nodejs_20
+            nodePackages.npm
```

Comment:
Still not sure that this is actually needed, since it comes bundled in the [`nodejs_20`](https://search.nixos.org/packages?channel=23.11&show=nodejs_20&from=0&size=50&sort=relevance&type=packages&query=nodejs) package.